composer.json

```diff
@@ -4,12 +4,13 @@
     "require": {
         "php": "^7.2",
         "ext-json": "*",
-        "beberlei/assert": "^2.9",
-        "composer/composer": "^1.6",
+        "beberlei/assert": "^3.2.0",
+        "composer/composer": "^1.8.5",
         "nikolaposa/version": "^3.1.0",
-        "roave/better-reflection": "^3.0",
-        "symfony/console": "^4.0",
-        "symfony/process": "^4.0"
+        "roave/better-reflection": "^3.4.0",
+        "symfony/console": "^4.2.7",
+        "symfony/process": "^4.2.7",
+        "thecodingmachine/safe": "^0.1.15"
```

Comment:
Yes, this is new, but for my own sanity I need to stop relying on the PHP internal functions when writing code.